examples/python/human_pose_tracking/README.md

```diff
@@ -16,9 +17,227 @@ channel = "main"
   <img src="https://static.rerun.io/human_pose_tracking/37d47fe7e3476513f9f58c38da515e2cd4a093f9/full.png" alt="">
 </picture>
 
-Use the [MediaPipe](https://google.github.io/mediapipe/) Pose solution to detect and track a human pose in video.
+# Overview
+
+Use the [MediaPipe](https://google.github.io/mediapipe/)  Pose Landmark Detection solutions to track human body pose in images and videos.
+
+Logging Details:
+
+1. Human Body Pose Landmarks as 2D Points:
+
+   - Specify connections between the points
+
+   - Extracts hand landmark points as 2D image coordinates.
+
+   - Logs the 2D points to the Rerun SDK.
+   
+
+2. Human Body Pose Landmarks as 3D Points:
+
+    - Specify connections between the points
+
+    - Detects body pose landmarks using MediaPipe solutions.
+
+    - Converts the detected body pose landmarks into 3D coordinates.
+
+    - Logs the 3D points to the Rerun SDK.
+
+# Run
 
 ```bash
 pip install -r examples/python/human_pose_tracking/requirements.txt
 python examples/python/human_pose_tracking/main.py
 ```
+# Usage
+
+CLI usage help is available using the `--help` option:
+
+```bash
+$ python examples/python/gesture_detection/main.py --help
+usage: main.py [-h] [--video {backflip,soccer}] [--dataset-dir DATASET_DIR] [--video-path VIDEO_PATH] [--no-segment] [--max-frame MAX_FRAME] [--headless] [--connect]
+               [--serve] [--addr ADDR] [--save SAVE] [-o]
+
+Uses the MediaPipe Pose solution to track a human pose in video.
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --video {backflip,soccer}
+                        The example video to run on.
+  --dataset-dir DATASET_DIR
+                        Directory to save example videos to.
+  --video-path VIDEO_PATH
+                        Full path to video to run on. Overrides `--video`.
+  --no-segment          Don t run person segmentation.
+  --max-frame MAX_FRAME
+                        Stop after processing this many frames. If not specified, will run until interrupted.
+  --headless            Don t show GUI
+  --connect             Connect to an external viewer
+  --serve               Serve a web viewer (WARNING: experimental feature)
+  --addr ADDR           Connect to this ip:port
+  --save SAVE           Save data to a .rrd file at this path
+  -o, --stdout          Log data to standard output, to be piped into a Rerun Viewer
+
+```
```

Comment:
I suggest we remove this section, because it will very likely code-rot (if we update the code we'll forget to update the README). Just tell the user to run `python examples/python/gesture_detection/main.py --help` and they can read it themselves